src/utils/line.ts

```diff
@@ -411,6 +390,48 @@ export const formatLinePageData = (
   return pageData;
 };
 
+export const formatUserPortfolioData = (
```

Comment:
Formatter function is calling another formatter function, which was returning a different data structure. 
Why do we need to format twice?